concent_api/utils/api_view.py

```diff
@@ -41,7 +41,7 @@ def wrapper(request, *args, **kwargs):
             message = None
         else:
             if request.content_type == '':
-                raise Http400('Content-Type is missing.')
+                return JsonResponse({'error': 'Content-Type is missing.'}, status = 400)
```

Comment:
There is a check above
```
if request.content_type not in ['application/octet-stream', '']:
             return JsonResponse({'error': 'Concent supports only application/octet-stream.'}, status = 415)
```
and below
```
else:
                 return JsonResponse({'error': "Concent supports only application/octet-stream."}, status = 400)
```
It looks like double checking the same thing, but with different status response. Please unify that.